rsync: build with the bundled zlib and popt, as hakchi v1.0.4-126's rsync was (it links neither
libz nor libpopt).

--- a/package/rsync/rsync.mk
+++ b/package/rsync/rsync.mk
@@ -8,10 +8,10 @@
 RSYNC_SITE = http://rsync.samba.org/ftp/rsync/src
 RSYNC_LICENSE = GPLv3+
 RSYNC_LICENSE_FILES = COPYING
-RSYNC_DEPENDENCIES = zlib popt
+RSYNC_DEPENDENCIES =
 RSYNC_CONF_OPTS = \
-	--with-included-zlib=no \
-	--with-included-popt=no
+	--with-included-zlib=yes \
+	--with-included-popt=yes
 
 ifeq ($(BR2_PACKAGE_ACL),y)
 RSYNC_DEPENDENCIES += acl
